returnn/tf/util/data.py

```diff
@@ -1201,13 +1246,15 @@ def copy_compatible_to(self, data, unbroadcast=False, except_feature=False,
       It currently does not check whether existing dims match.
     :param bool unbroadcast: if True, all broadcast axes (axes with dim 1) will be tiled such that they match
     :param bool except_feature: if unbroadcast, do not unbroadcast the feature dim
-    :param tf.Tensor|list[tf.Tensor|int]|tuple[tf.Tensor|int]|None data_dyn_shape:
-      For unbroadcast, if we do not want to rely on tf.shape(data.placeholder).
+    :param tf.Tensor|list[tf.Tensor|int]|tuple[tf.Tensor|int]|None data_dyn_shape: deprecated, will be ignored.
+      If unbroadcast=True, use the DimensionTags of `data` to specify the shape to unbroadcast to.
+      See :func:`copy_add_axis_by_dim_tag`.
     :param bool check_sparse:
     :param bool check_dtype:
     :returns: Data, might add broadcast dimensions
     :rtype: Data
     """
+    del data_dyn_shape  # ignored
```

Comment:
We better should better search for any code using this function, and clean that code up, instead of leaving this in.